src/style.css

```diff
@@ -1 +1,82 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, sans-serif;
```

Comment:
Consider adding a fallback font family for Avenir in case it fails to load. For example, 'font-family: Avenir, Arial, sans-serif;'. This ensures text is still readable with a default font.